packages/experimental/Icon/src/Icon.tsx

```diff
@@ -106,3 +101,11 @@ export const Icon = stagedComponent((props: IconProps) => {
 });
 
 export default Icon;
+
+function downgradeColor(color: ColorValue): string {
+  if (typeof color === 'string') {
+    return color as string;
+  }
+
+  return getCurrentAppearance(useFluentTheme().host.appearance, 'light') === 'dark' ? '#FFFFFF' : '#000000';
+}
```

Comment:
Maybe add a comment linking to our issue #728 ?

Reply:
Done.